README.md

```diff
@@ -21,14 +21,56 @@ Once you have implemented the `fm` program (and tested it to the best of your ab
 
 *What preprocessing data do you store in files, and how?*
 
+We store the preprocessed data in a .txt file with \n between each data point. For multiple sequnences in the same fasta file, one is stored after the other. We store the following:
+* The sequence header. This is needed for the output.
+* n, the length of the sequence. Can be deduced from the other data, but make sense to save, since we can then malloc with it. 
+* σ, alphabetsize. 
+* bwt of the converted sequence (where the chars are ints 0...σ-1). With commas between each char. We could same O and C, but they would take more space. 
+* SA. Comma seperated
+* The alphabet. No seperation chars, since each symbol is a unique char. 
+
+
 ## Insights you may have had while implementing the algorithm
+It takes 90% of the time to do the last 10% of the code, testing and writting the report. 
+
+Between this and the last handin, it has become part of the coding process to do most of the coding on one (Windows) machine, push the code and finish it on another (Linux). The code will be running fine on the first machine, but will be buggy on the other, which we will then fix. It is unclear which types of bugs is ignored, but it seems like the first machine is more forgiving with memory. 
```

Comment:
There are a bunch of libraries you can link with to get a tiny bit of memory checking. I don't have a lot of experience with them--I don't make mistakes, after all--but I have had some good experience with [efence](https://linux.die.net/man/3/efence) and [valgrind](https://valgrind.org/docs/manual/quick-start.html). As I recall, you just link with the library for the former and run the program with `valgrind` for the latter. (That is very slow, though, so don't run with large data).